src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,45 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_LOGIN_PASSWORD_LENGTH = 6;
+    private static final String MESSAGE_LENGTH_ERROR = "Your message length is less then 6";
+    private static final String USER_CANT_BE_NULL = "User cant be null";
+    private static final String CANNOT_BE_NULL_MESSAGE = "length can not be null";
+    private static final String YOU_ARE_NOT_OLD_ENOUGH = "Your age length is less then 18";
+    private static final String USER_ALREADY_EXIST = "This user already exist";
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        userCheck(user);
```

Comment:
Cool, but on the future, method names should refer to the verbs - `checkUser` or `validateUser` would be better